src/main/resources/edu/cornell/kfs/gl/cu-spring-gl.xml

```diff
@@ -209,6 +209,39 @@
        </property>
 	</bean>
 
+	<bean id="scrubberProcess-parentBean" class="org.kuali.kfs.gl.batch.service.impl.CuScrubberProcessImpl" abstract="true">
```

Comment:
Similar to what we're doing with the Poster, I'm overriding the base/parent Scrubber bean so that the numerous Scrubber child beans can all be overridden by updating just one bean.